articles/azure-monitor/insights/container-insights-faq.md

```diff
@@ -35,6 +41,14 @@ ContainerLog
 
 ```
 
+- option-2
+
+Re-enable collection for these fields for every container logline:
+
+If option-1 is not convenient due to query changes involved you have to change, you can re-enable collecting these fields by enabling the setting ```log_collection_settings.enrich_container_logs``` in the agent config map as described in the data collection configuration settings [here](./container-insights-agent-config.md)

```

Comment:
Vishwa, Looks like hyper link for configmap missing here. are you planning to add the agent config to this repo?

Reply:
config map is explained in the data collection configuration settings (which has everything thats linked here)

Reply:
i just added the new setting there so all are in 1 place

Reply:
> i just added the new setting there so all are in 1 place

lgtm.

Reply:
I suggest re-wording this sentence to - If the first option is not convenient due to ….